dev_requirements.txt

```diff
@@ -3,4 +3,4 @@ coveralls
 pytest>=2.9.0
 pytest-catchlog==1.2.2
 pytest-timeout==1.0.0
-https://github.com/ethereum/serpent/tarball/develop
```

Comment:
This should **NOT** be merged.  This is just here to get the python3 tests running until I can get https://github.com/ethereum/serpent/pull/102 merged.